### PR TITLE
CreateManifestTask checks that project.version is valid SLS version s…

### PR DESCRIPTION
**Pull request by uschi2000**
…tring

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/gradle-java-distribution/167)

### CODE REVIEW

**Comment by uschi2000** [comment]
FYI @aditkumar @briandorne

File: src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.groovy [hunk 1]
```diff
@@ -44,7 +45,11 @@ class CreateManifestTask extends DefaultTask {
 
     @Input
     public String getProjectVersion() {
-        return String.valueOf(project.version)
+        def stringVersion = String.valueOf(project.version)
+        if (!SlsProductVersions.isValidVersion(stringVersion)) {
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.groovy:49]: Probably worth a noisy-ish non-blocking warning if this isn't also an orderable version.

**Review by markelliot — APPROVED** [review]
Your choice on solving my one comment.

File: src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.groovy (hunk 1 above)
Comment (uschi2000) [reply src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.groovy:49]: done

**Pull request merged**